.eslintrc.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
         'no-underscore-dangle': 'off',
         'promise/always-return': 'off',
         'promise/catch-or-return': 'off',
+        'promise/no-nesting': 'off',
```

Comment:
Do we definitely want to turn this off by default, or are nested promises used sparingly enough that adding a comment to ignore this rule on a case by case basis would work?

Reply:
Yeah in the context of the whole application we do not like nesting promises. In cypress the context changes as we HAVE to nest promises as that is how their chainables work, so we modified our lint to ignore those rules in cypress.